internal/cri/annotations/annotations.go

```diff
@@ -106,19 +106,22 @@ func DefaultCRIAnnotations(
 		customopts.WithAnnotation(SandboxNamespace, config.GetMetadata().GetNamespace()),
 		customopts.WithAnnotation(SandboxUID, config.GetMetadata().GetUid()),
 		customopts.WithAnnotation(SandboxName, config.GetMetadata().GetName()),
+		customopts.WithAnnotation(ImageName, imageName),
 	}
 	ctrType := ContainerTypeContainer
 	if sandbox {
 		ctrType = ContainerTypeSandbox
 		// Sandbox log dir only gets passed for sandboxes, the other metadata always
 		// gets sent however.
-		opts = append(opts, customopts.WithAnnotation(SandboxLogDir, config.GetLogDirectory()))
+		opts = append(
```

Comment:
pls revert format change

Reply:
sandbox log dir and sandbox image name all get passed. Do I need to revert the format now?